worldeditadditions/utils/tables/init.lua

```diff
@@ -9,12 +9,14 @@
 -- extensive collection of functions :P
 
 -- TODO: Refactor into its own worldeditadditions.tables namespace.
+worldeditadditions.tables = {}
 
+dofile(worldeditadditions.modpath.."/utils/tables/sets.lua")
 dofile(worldeditadditions.modpath.."/utils/tables/shallowcopy.lua")
 dofile(worldeditadditions.modpath.."/utils/tables/table_apply.lua")
-dofile(worldeditadditions.modpath.."/utils/tables/table_unpack.lua")
+dofile(worldeditadditions.modpath.."/utils/tables/table_filter.lua")
 dofile(worldeditadditions.modpath.."/utils/tables/table_get_last.lua")
-dofile(worldeditadditions.modpath.."/utils/tables/table_tostring.lua")
 dofile(worldeditadditions.modpath.."/utils/tables/table_map.lua")
-dofile(worldeditadditions.modpath.."/utils/tables/table_filter.lua")
```

Comment:
Why remove these dofile() calls?

Reply:
I didn't them, I simply alphabetized them.

Reply:
Ah, I see now! Oops, didn't see that :P